src/app/produkt/[id]/ProductVariants.tsx

```diff
@@ -411,7 +427,9 @@ const ProductVariants = ({ product }: { product: Product }) => {
                     </Button>
                   </Table.HeaderCell>
                 ) : (
-                  <Table.HeaderCell>HMS-nummer</Table.HeaderCell>
+                  <Table.HeaderCell style={{
+                    zIndex: "2 !important",
```

Comment:
Fix av sticky-hms nr. TH havna bak kolonner uten stickyness.

Reply:
Hm, denne blir ikkje alltid satt, men overrida av reglen:  .variants-table tr>th:first-child 
som feilaktig gjer den z-index: 1 istaden for 2.